### ISSUE TITLE
This sentence is hard to follow. 

### ISSUE THREAD

**Issue by dig-comments** [usnistgov/800-63-3#846]
**Organization Name (N/A, if individual)**: National Security Agency

**Organization Type**: 1
 
**Document (63-3, 63A, 63B, or 63C)**: 800-63B

**Reference (Include section and paragraph number)**: 2, 5
 
**Comment (Include rationale for comment)**:  This sentence is hard to follow.
 
Suggest rewording as: "Stronger authentication (a higher AAL) requires attackers to have better capabilities and expend greater resources in order to successfully subvert the authentication process. These higher AALs can effectively reduce the risk of these types of attacks." Or work in the definition from 800-63-3 of authentication  error:" a true authentication error (a false claimant using a credential that is not rightfully theirs)"

Original text:
"Stronger authentication (a higher AAL) requires a higher level of capabilities or resources on the part of an attacker in order to successfully authenticate, effectively reducing the risk of an authentication error."

**Suggested Change**: Make suggested change.
 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jimfenton** [comment]
I like your wording; thanks.
